### PR TITLE
Companion for substrate/pull/7215

### PR DESCRIPTION
**Pull request by kianenigma**

### CODE REVIEW

File: runtime/westend/src/lib.rs [hunk 1]
```diff
@@ -32,7 +32,7 @@ use runtime_common::{
 	impls::{CurrencyToVoteHandler, ToAuthor},
 	BlockHashCount, MaximumBlockWeight, AvailableBlockRatio, MaximumBlockLength,
 	BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight, MaximumExtrinsicWeight,
-	ParachainSessionKeyPlaceholder,
+	ParachainSessionKeyPlaceholder, AVERAGE_ON_INITIALIZE_WEIGHT,
```
Comment (thiolliere) [inline runtime/westend/src/lib.rs:35]: unused import ?
```suggestion
	ParachainSessionKeyPlaceholder,
```

File: runtime/polkadot/src/lib.rs [hunk 2]
```diff
@@ -25,7 +25,7 @@ use runtime_common::{
 	impls::{CurrencyToVoteHandler, ToAuthor},
 	NegativeImbalance, BlockHashCount, MaximumBlockWeight, AvailableBlockRatio,
 	MaximumBlockLength, BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight,
-	MaximumExtrinsicWeight, purchase, ParachainSessionKeyPlaceholder,
+	MaximumExtrinsicWeight, purchase, ParachainSessionKeyPlaceholder, AVERAGE_ON_INITIALIZE_WEIGHT,
```
Comment (thiolliere) [inline runtime/polkadot/src/lib.rs:28]: unused import ?
```suggestion
	MaximumExtrinsicWeight, purchase, ParachainSessionKeyPlaceholder,
```

File: runtime/kusama/src/lib.rs [hunk 3]
```diff
@@ -33,7 +33,7 @@ use runtime_common::{
 	impls::{CurrencyToVoteHandler, ToAuthor},
 	NegativeImbalance, BlockHashCount, MaximumBlockWeight, AvailableBlockRatio,
 	MaximumBlockLength, BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight,
-	MaximumExtrinsicWeight, ParachainSessionKeyPlaceholder,
+	MaximumExtrinsicWeight, ParachainSessionKeyPlaceholder, AVERAGE_ON_INITIALIZE_WEIGHT,
```
Comment (thiolliere) [inline runtime/kusama/src/lib.rs:36]: unused import ?
```suggestion
	MaximumExtrinsicWeight, ParachainSessionKeyPlaceholder,
```

**Review by thiolliere — APPROVED** [review]

**Comment by kianenigma** [comment]
bot merge

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Pull request merged**